test_util/rewriteEclFile.cpp

```diff
@@ -0,0 +1,102 @@
+/*
+  Copyright 2019 Equinor ASA.
+
+  This file is part of the Open Porous Media project (OPM).
+
+  OPM is free software: you can redistribute it and/or modify
+  it under the terms of the GNU General Public License as published by
+  the Free Software Foundation, either version 3 of the License, or
+  (at your option) any later version.
+
+  OPM is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+
+  You should have received a copy of the GNU General Public License
+  along with OPM.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#include <cmath>
+#include <iomanip>
+#include <iostream>
+#include <tuple>
+#include <getopt.h>
+#include <sstream>
+#include <stdexcept>
+
+#include <opm/io/eclipse/EclFile.hpp>
+#include <opm/io/eclipse/EclOutput.hpp>
+
+
+static void printHelp() {
+
+    std::cout << "\rewriteEclFile needs a minimum of one arguments which is the input file name.  \n"
+              << "\nIn addition, the program takes these options (which must be given before the arguments):\n\n"
+              << "-h Print help and exit.\n\n";
+}
+
+int main(int argc, char **argv) {
```

Comment:
This program looks like a advanced `cp` - maybe not something to check in?

Reply:
The program opens one file specified on the command line. All arrays are loaded and content is rewritten to a new file. If everything went well, the input and output file should be identical. 

As we talked about earlier today, I have added the actual comparison into the program. Thanks, that was a good suggestion. Notice that I have renamed the program to _rewrite_and_compare_.